.travis.yml

```diff
@@ -17,13 +17,20 @@ script:
   - ./vendor/bin/phpunit
   # Integration test
   - mkdir web && echo "Hello" > web/index.html
-  - bin/ppm start --workers=1 --bridge=StaticBridge --static-directory=web --max-requests=1 -v &
+  - bin/ppm start --workers=1 --bridge=PHPPM\\Tests\\TestBridge --static-directory=web --max-requests=1 -v &
```

Comment:
Please make this a separate test- I'd really like to see if the static bridge itself does anything sensible